Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Setup a Python virtual environment
+FROM python:3.9.13-slim-bullseye as python-builder
+RUN apt-get update && apt upgrade -y && pip install pipenv
+WORKDIR /opt/app
+COPY ./requirements.txt .
+ENV PIPENV_VENV_IN_PROJECT=1
+RUN pipenv install --three
+
+# Build the transifex-cli go static binary
+FROM golang:1-bullseye as go-builder
+WORKDIR /opt/app
+RUN git clone https://github.com/transifex/cli . && make build
```

Comment:
what if their master is currently broken? Can we install either a fixed version or the latest stable version?

Reply:
Thanks.